### PR TITLE
add custom links option in sidebar nav

### PR DESCRIPTION
**Pull request by sinisaos**
Related to #307.

### CODE REVIEW

File: piccolo_admin/endpoints.py [hunk 1]
```diff
@@ -401,6 +401,7 @@ def __init__(
         translations: t.List[Translation] = None,
         allowed_hosts: t.Sequence[str] = [],
         debug: bool = False,
+        sidebar_links: t.List[t.Tuple[str, str]] = [],
```
Comment (dantownsend) [inline piccolo_admin/endpoints.py:404]: I think this was my fault, suggesting tuples. But if we just the user pass in a dict instead, we no longer need `get_sidebar_links`.

```
sidebar_links={
    "Top Movies": "/movies/?order=-popularity
```

**Review by dantownsend — COMMENTED** [review]
This looks great - thanks!

Sorry it took me so long to review. Just one suggestion. And then we'll add docs and release it.

**Comment by sinisaos** [comment]
@dantownsend I've updated the `sidebar_links` docstring and added an example. Is that enough because `create_admin` docs don't have a separate section for parameters, they are from the source code docstring?

File: piccolo_admin/endpoints.py (hunk 1 above)
Comment (dantownsend) [reply piccolo_admin/endpoints.py]: > Am I missing something?

No, you're right - I just meant we didn't need the bit that mapped tuples into dictionaries any more, as they're already dictionaries.

**Comment by sinisaos** [comment]
> I see what you're saying. It might be nice to have a screen grab, so people get a better idea of what it does.
> 
> Based on the current docs, we would probably need a new page called `Sidebar Links` or something.

I agree. I can do it, but I think it would be better if you do it, because I see that almost all screenshots are from Safari web browser (and I don't have a Mac).

**Comment by dantownsend** [comment]
> I agree. I can do it, but I think it would be better if you do it, because I see that almost all screenshots are from Safari and Mac (and I don't have a Mac).

OK, no problem.

Something like this might be OK, which doesn't have the border or address bar, so is browser / OS agnostic:

https://piccolo-admin.readthedocs.io/en/latest/_images/forms_sidebar.png

**Comment by sinisaos** [comment]
If that's the case, I can try to make docs with screenshots and make a new commit.

**Comment by dantownsend** [comment]
> If that's the case, I can try to make docs with screenshots and make a new commit.

Thanks!

**Comment by dantownsend** [comment]
Made some minor changes:

* I rearranged the docs a bit - gave `Side Bar Links` its own page.
* Using normal `a` tags instead of `router-link`, so external links also work.

**Comment by dantownsend** [comment]
@sinisaos If you're happy with the changes I made, will merge it in tomorrow.

**Comment by sinisaos** [comment]
@dantownsend Your changes look great. Thanks. Feel free to merge it.

**Pull request merged**